src/Config/packages.aget

```diff
@@ -2,13 +2,13 @@
   "nuget": {
     "references": {
       "DynamicLanguageRuntime": "1.2.2",
-      "DynamoVisualProgramming.Core": "3.3.0.6316",
-      "DynamoVisualProgramming.DynamoCoreNodes": "3.3.0.6316",
-      "DynamoVisualProgramming.DynamoServices": "3.3.0.6316",
-      "DynamoVisualProgramming.Tests": "3.3.0.6316",
-      "DynamoVisualProgramming.WpfUILibrary": "3.3.0.6316",
-      "DynamoVisualProgramming.ZeroTouchLibrary": "3.3.0.6316",
-      "DynamoVisualProgramming.DynamoSamples": "3.3.0.6316",
+      "DynamoVisualProgramming.Core": "3.4.0-beta6616",
```

Comment:
In order to merge in Revit we need to use a build that has retention "keep" in artifactory. This build currently has retention policy 30 ( days ).

Reply:
Good catch! I changed the policy to keep.